prompts/Dutch (Netherlands)_nl-NL/0000000001_0300000050_General.txt

```diff
@@ -45,36 +45,36 @@
 0000000045	Dat is echt leuk voor de hele familie, want niet alleen ben je lekker in de buitenlucht met elkaar bezig maar je komt ook tal van hindernissen tegen.
 0000000046	Het is niet leuk voor ze als pap en mams alle puzzels oplossen en zij er maar voor spek en bonen bijzitten.
 0000000047	Het is één van de klassieke uitjes voor de hele familie die nog altijd razend populair zijn.
-0000000048	Met een groep van maximaal tien personen zul je plaatsnemen op een rubberen ribboot, die zeer hoge snelheden kan behalen.
+0000000048	Met een groep van maximaal tien personen zul je plaatsnemen op een rubberen roeiboot, die zeer hoge snelheden kan behalen.
```

Comment:
Unfortunately, `roeiboot` (rowing boat) isn't correct in this context. It wouldn't be the boat type that will get to high speeds (as the text writes).

Reply:
Ah, learned something new!